readme.md

```diff
@@ -132,6 +132,68 @@ This is where your stories will be registered and where the Storybook configs wi
+
+### Some Highlights of MST
+
+MST is...
+
+- Intuitive
+  - With concepts like `props` and `actions`, it feels familiar for React developers
+  - Updating your data means calling functions on objects, rather than dispatching actions.
+  - Feels similar to relational databases, with concepts like `identifiers` (primary keys), `references` (foreign keys), and `views` (calculated fields)
+- Streamlined
+  - No more `actionTypes`, `actionCreators`, or `reducers`
+  - You don't have to declare your usage intentions with `mapStateToProps`, they are inferred
+  - Side-effects are built in, no need for 3rd party libraries like `redux-saga`, `redux-observable`, or `redux-thunk`
+  - Immutability is built-in - no need for `immutable.js` or `seamless-immutable`
+  - `types.compose` and `model.extend` allow for easy code-sharing of common patterns
+- More than state management
+  - Lifecycle hooks like `afterCreate`, `preProcessSnapshot`, and `beforeDestroy` let you have control over your data at various points in it's lifecycle
+  - Dependency injection with `getEnv` allows you to reference your environment (like API or other services)
+- Performant
+  - Round-trip store writes are much faster
+  - Computed values (views) are only calculated when needed
+
+### Downsides
+
+We also recognize no state management solution is perfect. MST has some known downfalls:
+
+- Integration with TypeScript is clunky at times. MST's own typing system is sometimes at odds with what TypeScript wants
+- `mobx` and `mobx-state-tree` both tend to have quite a lot of "magic" or "sorcery" that makes issues hard to debug because you don't always have a clear picture of what's happening
+- The user base is small, so finding help on GitHub or Stack overflow is difficult
+- Fatal errors are sometimes too-easily triggered and error messages can be verbose and hard to grok
+- The API has a huge surface area and the docs are very technical and unfriendly
+
+## Learning MobX State Tree
+
+MobX and MobX State Tree can be a bit confusing if you're coming from Redux, so here are a few of our favorite resources to learn the basics:
+
+- Be sure to check out the official [Getting Started](https://github.com/mobxjs/mobx-state-tree/blob/master/docs/getting-started.md) guide for MobX State Tree.
+
+- There is also a free [egghead.io course](https://egghead.io/courses/manage-application-state-with-mobx-state-tree).
+
+- For a great explanation and walkthrough of the basics, check out [State Management with MobX State Tree](https://medium.com/react-native-training/state-management-with-mobx-state-tree-373f9f2dc68a) by React Native Training.
+
+- And for more in-depth reading, the [official documentation](https://github.com/mobxjs/mobx-state-tree/blob/master/README.md) is a great resource
+
```

Comment:
We should probably mention the spectrum channel and our own community slack as well: https://spectrum.chat/mobx-state-tree

Reply:
I actually didn't know about that!